### PR TITLE
Fix markdown issues

### LINKED ISSUE #3746: Markdown Fenced PHP Code Block is Not Syntax Highlighted
When creating a code fence, the block doesn't get colored as code (stays white), but it does display as code in the preview.

    ```php
    <?php
    //Hello World
    echo ('Hello World!');
    ?>
    ```

If I tab the code with 4 spaces it does color it properly.

Here is an example:

screenshot from 2016-03-05 16-45-51

### LINKED ISSUE #6113: Bracket auto-complete in markdown
- VSCode Version:1.1.0-alpha
- OS Version: Windows 10

Steps to Reproduce:

1. Open a markdown file and add a code block.
2. Type a function, string, etc that has an opening bracket or string

Regardless of the language specified for the code block, bracket completion is not supported.

### PR DESCRIPTION
**Pull request by rebornix**
The markdown plist language include section is generated by scripts I wrote, no idea why PHP was missing in the beginning. But now I bring it back. Fixes #3746 .

The second commit is adding brackets auto-closing in markdown. Fixes #6113 .

### CODE REVIEW

File: extensions/markdown/syntaxes/markdown.tmLanguage [hunk 1]
```diff
@@ -658,6 +662,22 @@
...
+					<string>(^|\G)\s*([`~]{3,})\s*(php|php3|php4|php5|phpt|phtml|aw|ctp)\s*$</string>
+					<key>name</key>
+					<string>markup.fenced_code.block.markdown</string>
+					<key>end</key>
+					<string>(^|\G)(\2)\n</string>
+					<key>patterns</key>
+					<array>
+						<dict>
+							<key>include</key>
+							<string>text.html.php</string>
```
Comment (rebornix) [inline extensions/markdown/syntaxes/markdown.tmLanguage]: It's the same as the scopeName inside php syntax file, but currently the inline highlighting doesn't work. Will look into it later on.

**Comment by bpasero** [comment]
@rebornix please feel free to assign the PR to someone like @kieferrm if you want a quick turnaround 👍

**Comment by bpasero** [comment]
@rebornix if this is for June, please check it in by today.

**Comment by rebornix** [comment]
@bpasero Kai told me he will merge this last time we synced so it's up to him which release he wants this change in, I don't get the privilege to check in yet.

**Comment by bpasero** [comment]
Ok maybe you ask him in person, I am not sure he checks GH notifications.

**Pull request merged**